app/configurator/components/chart-type-selector.tsx

```diff
@@ -192,6 +185,21 @@ export const ChartTypeSelector = ({
             </Flex>
           )}
         </ControlSectionContent>
+
+        {possibleFiltersFetching ? (
+          <CircularProgress
+            color="primary"
+            size={12}
+            sx={{
+              position: "absolute",
+              bottom: 48,
+              right: 48,
+              color: "hint.main",
+              display: "inline-block",
+              ml: 1,
+            }}
+          />
+        ) : null}
```

Comment:
I wonder in fact why we have the spinner for possible filters fetching here ? It's already shown in the fitlers control section no ?

Reply:
Done!